src/adapter-types/src/connection.rs

```diff
@@ -16,3 +17,45 @@ pub type ConnectionIdType = u32;
 
 /// An abstraction allowing us to name different connections.
 pub type ConnectionId = IdHandle<ConnectionIdType, IdAllocatorInnerBitSet>;
+
+/// Number of bits the org id is offset into a connection id.
+pub const ORG_ID_OFFSET: usize = 19;
+
+/// Extracts the lower 12 bits from an org id
```

Comment:
What do you think about adding a bit more context here?

```suggestion
/// Extracts the lower 12 bits from an org id. These are later used as the [31, 20] bits of a connection id to help route cancellation requests.
```

Reply:
Added.